### DISCUSSION TITLE
Explain decoder

### DISCUSSION THREAD

**Discussion by ShaniCohen** [pyg-team/pytorch_geometric#9422] [Q&A]
I have built a GNN model for link prediction in a bipartite graph.
My model is composed of an encoder & decoder, similar to your [(example)](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/explain/captum_explainer_hetero_link.py) but with a concatenation of additional features fed to the decoder.
I want to use PyG Explainer (or any other PyG class) to explain the contributions of the decoder's input features (or explain the decoder). Would you happen to have any support for this?
BR

**Comment by rusty1s** [comment]
If you just want to explain the decoder (with no GNN involved), you can simply use any explanation tool from `captum`. Does that work for your case?
